File: fizzbuzz.R
rm(list=ls())
# fizzbuzz

# For the numbers 1 to 100, print “fizz” if the number is divisible by 3, 
# “buzz” if the number is divisible by 5, 
# “fizzbuzz” if the number is divisible by both 5 and 3,
# and otherwise print the number.

fizzbuzz <- function(n) {
  if ('&'(n %% 3 == 0, n %% 5 == 0)) {
    return("fizzbuzz")
  } else if (n %% 3 == 0) {
    return("fizz")
  } else if (n %% 5 == 0) {
    return("buzz")
  } else return(n)
}

sapply(1:100, fizzbuzz)

File: fizzbuzz.c
#include <stdio.h>

int main(void) {

	int i;

	for (i = 1; i <= 100;i++) {
		if (i % 3 == 0 && i % 5 == 0)
			printf("%d is fizzbuzz\n", i);
		else if (i % 3 == 0)
			printf("%d is fizz\n", i);
		else if (i % 5 == 0)
			printf("%d is buzz\n", i);
		else
			printf("%d\n", i);
	}
}

